lighthouse-core/test/audits/legacy-javascript-test.js

```diff
@@ -28,6 +28,7 @@ const createArtifacts = (scripts) => {
       };
       return acc;
     }, {}),
+    SourceMaps: [],
```

Comment:
could we a unit test or two in here to exercise the deduplication piece and any other flow that isn't currently covered by the bash script?

Reply:
done